jina/drivers/multimodal.py

```diff
@@ -83,11 +68,11 @@ def _apply_all(
 
         valid_docs = []
         for doc in docs:
-            doc_content = _extract_modalities_from_document(doc)
+            doc_content = doc.extract_content()
```

Comment:
This naming is ambiguous, because it seems that it only extracts the content for a document (and this extracts content from chunks grouped by modality)